.github/scripts/common.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+if [ "${RUNNER_OS}" = "Windows" ] ; then
+	ext=".exe"
+else
+	ext=''
+fi
+
+# Colors
+RED="0;31"
+LT_BROWN="1;33"
+LT_BLUE="1;34"
+
+ecabal() {
+	cabal "$@"
+}
+
+sync_from() {
```

Comment:
Functions like these seem to prefer globals over parameters, are these globals constants? If yes, ignore this comment, otherwise, I think I'd prefer explicit parameters.

Reply:
The globals are secrets or variables from the workflow file.